ci/checksizes.sh

```diff
@@ -37,7 +37,7 @@ REVERSI_OPT_GZ_SIZE=$(stat '-c%s' "$REVERSI_OPT.gz")
 
 case $FULLVER in
   2.10.2)
-    REVERSI_PREOPT_EXPECTEDSIZE=532000
+    REVERSI_PREOPT_EXPECTEDSIZE=533000
```

Comment:
The small increase in code size comes from the indirection through `runtime.toScalaVarArgs`. Even though it is inlined, the argument must be extracted in a temp var by the optimizer to preserve evaluation order (or so it thinks) which slightly increase code size.